widgets/tabs/js/tabs.js

```diff
@@ -121,7 +121,9 @@ jQuery( function ( $ ) {
 					if ( window.location.hash ) {
 						var anchors = window.location.hash.replace( '#', '' ).split( ',' );
 						anchors.forEach( function ( anchor ) {
-							var tab = $tabs.filter( '[data-anchor="' + anchor + '"]' );
+							var tab = $tabs.filter( function ( index, element ) {
+								return decodeURI( anchor ) === decodeURI( $( element ).data( 'anchor' ) );
+							} );
 							if ( tab ) {
```

Comment:
The return value of the jQuery filter method is a jQuery object, so `tab` will always be truthy. A more accurate test here is perhaps `if ( tab.length > 0 )`.